util/genbuild_h/genbuild_h.sh

```diff
@@ -66,6 +66,10 @@ printf "#define COREBOOT_VERSION %s\n" "\"$KERNELVERSION\""
 
 #See if the build is running in a git repo and the git command is available
 printf "/* timesource: $TIMESOURCE */\n"
+
+printf "#define DASHARO_VERSION \"%s\"\n" `git describe --tags --match "asus_kgpe-d16_v[0-9]*.[0-9]*.[0-9]*" | sed 's/asus\_kgpe\-d16\_//' | sed 's/\(v[0-9]\.[0-9]\.[0-9]\+\).*/\1/'`
```

Comment:
@mkopec @krystian-hebel TBH this not my code, do you have snippet that should work, so I can replace and test?

Reply:
I see that I have dell_optiplex_9010 in #195

Reply:
yes.. however, on #195 we have COREBOOT_MAJOR_VERSION but here is the DASHARO_VERSION. This is inconsistent

Reply:
Another problem we were trying to solve was to not use the CONFIG_LOCALVERSION actually, because setting it to e.g. v1.0.5 would mean that there can be multiple v1.0.5 binaries, not necessarily built from v1.0.5 tag. This way we ensured only tags can produce a meaningful DASHARO_VERSION.